src/NosCore.Controllers/CommandPacketController.cs

```diff
@@ -71,6 +72,33 @@ public CommandPacketController()
         {
         }
 
+        [UsedImplicitly]
+        public void Kick(KickPacket kickPacket)
+        {
+            var servers = WebApiAccess.Instance.Get<List<ChannelInfo>>("api/channel");
```

Comment:
This entire method doesn't make sense... You need to check if the account is connected on each channel. Not only on the last one

Then, if you don't use a connectedaccount in when you delete on the WebApi, don't use one